Package.swift

```diff
@@ -0,0 +1,23 @@
+// swift-tools-version: 5.8
+// The swift-tools-version declares the minimum version of Swift required to build this package.
```

Comment:
Do we need this comment?

Reply:
no this comment is auto generated and not needed